tests/integration/lambdas/lambda_integration.py

```diff
@@ -90,7 +90,7 @@ def handler(event, context):
             forwarding_target = ddb_new_image['data'][MSG_BODY_MESSAGE_TARGET]
             target_name = forwarding_target.split(':')[-1]
             if forwarding_target.startswith('kinesis:'):
-                ddb_new_image['data'][MSG_BODY_MESSAGE_TARGET] = 's3:/test_chain_result'
+                ddb_new_image['data'][MSG_BODY_MESSAGE_TARGET] = 's3:test_chain_result'
```

Comment:
Out of curiosity, shouldn't this be a proper URL like `s3://test_chain_result`, instead of `s3:test_chain_result`?